.travis.yml

```diff
@@ -1,3 +1,45 @@
+dist: bionic
+
 language: node_js
 node_js:
-  - '0.10'
+  - 14.16.0
+
+cache:
+  yarn: true
+
+env:
+  global:
+    - PATH="$HOME/.yarn/bin:$PATH"
+    - YARN_VERSION="1.9.4"
+
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version "${YARN_VERSION}"
+
+install:
+  - yarn install --frozen-lockfile
+
+jobs:
+  include:
+    # Install dependencies and save them to the Travis cache
+    - stage: "Prepare cache"
+      # Don't run `script`, just install dependencies
+      before_script: skip
+      script: skip
+    # Run tests
+    - stage: "Test"
+      name: "Lint"
+      script:
+      # Lint this package using this package. How meta.
+      - yarn link
+      - yarn link crashpad
+      - yarn run lint
+    - stage: "Publish to registry"
```

Comment:
Thanks for adding! This won't actually work without an NPM token configured though. Could you ask Platform Eng in #help-platform-engineering for an NPM automation token for a public package (crashpad)? Then follow these steps: https://goodeggs.atlassian.net/wiki/spaces/ENG/pages/459833453/Using+NPM+and+Yarn#How-to-configure-Travis-CI-to-automatically-publish-to-the-public-registry.